ooni/scripts/oonideckgen.py

```diff
@@ -0,0 +1,138 @@
+from __future__ import print_function
+
+import errno
+import os
+import sys
+
+from twisted.internet import defer, task
+from twisted.python import usage
+
+from ooni.otime import prettyDateNowUTC
+from ooni import errors
+from ooni.geoip import probe_ip
+from ooni.resources import check_for_update
+from ooni.settings import config
+from ooni.deck import NGDeck
+
+__version__ = "1.0.0"
+
+class Options(usage.Options):
+    synopsis = """%s [options]
+    """ % sys.argv[0]
+
+    optParameters = [
+        ["country-code", "c", None,
+         "Specify the two letter country code for which we should "
+         "generate the deck."],
+        ["collector", None, None, "Specify a custom collector to use when "
+                                  "submitting reports"],
+        ["bouncer", None, None, "Specify a custom bouncer to use"],
+        ["output", "o", None,
+         "Specify the path where we should be writing the deck to."]
+    ]
+
+    def opt_version(self):
+        print("oonideckgen version: " % __version__)
```

Comment:
Yes I agree, maybe I will just change all the subcommands to follow the same versioning of ooniprobe itself.